.vsts-dotnet.yml

```
@@ -42,11 +42,13 @@ stages:
   displayName: Build
 
   jobs:
-  - template: /eng/common/templates/job/onelocbuild.yml
-    parameters:
-      CreatePr: false
-      LclSource: lclFilesfromPackage
-      LclPackageId: 'LCL-JUNO-PROD-MSBUILD'
+  - {{ if eq(variables['Build.SourceBranch'], 'refs/heads/main') }}:
+    - template: /eng/common/templates/job/onelocbuild.yml
+      parameters:
+        CreatePr: false
+        LclSource: lclFilesfromPackage
+        LclPackageId: 'LCL-JUNO-PROD-MSBUILD'
+        MirrorRepo: {{msbuild}}
```

another doc change I should make -- should just be `msbuild`. The `{{}}` is for placeholders within the doc... but obviously that's going to be confusing since they're also used in YAML. 🤦‍♀️

```suggestion
        MirrorRepo: msbuild
```